app/scripts/constants.js

```diff
@@ -676,5 +676,10 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     'openshift-',
     'kubernetes-',
     'kube-'
-  ]
+  ],
+  MOBILE_CLIENT_VERSION: {
```

Comment:
These seem to be kept in https://github.com/openshift/origin-web-common/blob/master/src/services/apiService.js and https://github.com/openshift/origin-web-common/blob/676e5bef4da0e6fbf1c9237214aea2ad71e31a15/src/constants/apiPreferredVersions.js

It would mean a change against web-common but we might look at doing that too.

Reply:
https://github.com/openshift/origin-web-common/pull/320